### DISCUSSION TITLE
New midround antag: Facehugger Parasite

### DISCUSSION THREAD

**Discussion by Shaddap1** [space-wizards/space-station-14#33294] [Ideaguys]
In complete honesty I have never once played colonial marines or RMC.  This is my ground-up approach to facehugger for Wizden.

The Facehugger Parasite (final name TBD) is an individually weak midround swarming antag that requires crew cooperation to stop.

May spawn in maints or suddenly erupt from station animals (cows, chickens, mice, monkeys). 
Between 2-4 spawn at a time
Has 50 health, 25% blunt resist, -50% heat resist resist.  Can survive in space for a medium duration, pressure, o2 loss, and cold resist values TBD. 

Has one ability: Parasitic Leap (8 second cooldown)

- The parasite leaps forward 2 tiles.  If it hits a humanoid or animal target, it clamps onto its face, blinding it.  The target may still move/interact normally.  The parasite may leap onto crit or dead bodies.

- If the target is wearing a face-covering helmet (any EVA helmet, riot helmet, etc) it takes 10 seconds to dissolve the helmet, unequipping it.  Bio-hazard hoods are specifically formulate to resist this acid and take 20 seconds to dissolve.

- Once the helmet is destroyed, or if target was not wearing a helmet, the helmet slot is replaced with the parasite, and the parasite begins injecting an egg and a toxin.  (The parasite is able to inject the egg and toxin through masks).  The target is additionally "muzzled," and can only communicate verbally through muffled screams ("MMMPH! MMMHPH!!")

- The parasite cannot be attacked as an individual entity once it has "equipped" itself to a player's headslot.  The parasite takes 1/3 of any damage dealt to the player it has attached to.  

- Once the parasite has "equipped" itself to a player head slot, the target is dealt 2 toxin and 5 stamina damage per second for 20 seconds

- After 20 seconds, the target is stunned.  A new parasite erups from their chest, screeching, and dealing 50 slash damage.  The target is stunned for 6 seconds, almost crit, and badly bleeding  (90 total health damage, 100 stamina damage (stamcrit)).  The original parasite detaches and has leap has an 8 second cooldown.

- The target may not remove the parasite by themselves.  The parasite must be killed through self-damage, being clobbered by someone else, or removed by another crew member.  
- A humanoid player can pull off the parasite off of someone else in 10 seconds with their bare hands or it can be removed in 5 seconds with a crobar by a humanoid or borg.
- Killing the parasite while it is on the head slot of a player requires attacking the player, with the facehugger receiving 1/3 of the damage dealt.

- If a parasite is successfully removed, it is stunned for 1 second and may not leap again for 8 seconds.

- Parasites cannot open doors.  They may squeeze through unbolted doors, taking 10 seconds to do so, and making a door creaking sound similar to dragon devouring walls/doors. 

- Parasites can be either AI controlled or player controlled as a ghost role, like dragon's space carp.

- Parasites can drink blood to satiate hunger, thirst, and heal.

MAJOR ISSUE, TBD: How to properly prevent facehuggers from infesting the same corpse over and over.  Egg only spawns if the target has above a certain blood volume percentage threshold?

Gameplay will involve facehuggers seeking to multiply as quicky as possible.  They can consume dead and living animals crew to do so. They are individually weak and can be killed reasonably with knives and more easily with welders.  They are extremely vulnerable to lasers.

The crew is incentivized to immediately secure and exterminate all animals, especially maints mice.  They are also incentivized to travel in groups.  If you get facehugged, you need to seek out fellow crew to help you as quickly as possible.  Attacking yourself in the face may allow you to escape one facehugger, but it is not sustainable.

Security is incentivized to equip lasers, form extermination squads, and protect the medical department.
Medical is incentivized to equip and distribute biohazard clothing and maintain a secure medbay.

This antag is designed as part of a larger xeno-style gamemode, where an alien Queen will arrive on station, lay facehugger eggs which must be protected, and consume crew to continue laying more eggs.

**Comment by Boaz1111** [comment]
This should be turned into a design document

**Comment by AboveDeceased** [comment]
this is big if this gets added but you should really make a design document.
